calculator.js

```diff
@@ -1 +1,180 @@
-console.log('start')
\ No newline at end of file
+const numbers = document.querySelectorAll('.number')
+const mainResult = document.querySelector('.result')
+const operators = document.querySelectorAll('.operator')
+const equal = document.querySelectorAll('.equal')
+const deleteEqual = document.querySelectorAll('.deleteEqual')
+const dot = document.querySelectorAll('.dot')
+const deleteLast = document.querySelectorAll('.deleteLast')
+
+let result = ''
+let number1 = ''
+let number2 = ''
```

Comment:
I can be wrong, but I think that in such calculator we only need result and one number and second is unnecessary. Did you check? I think we talked about that by phone.